admins/pageflow/accounts.rb

```diff
@@ -3,14 +3,24 @@ module Pageflow
     menu :priority => 3
 
     config.batch_actions = false
-    config.clear_sidebar_sections!
 
     index do
       column :name do |account|
         link_to account.name, admin_account_path(account)
       end
+      column I18n.t('pageflow.admin.accounts.entries_count') do |account|
```

Comment:
Add translation key `activerecord.attributes."pageflow/account".entries_count` instead. That way it is picked up by ActiveAdmin automatically if you use `column :entries_count`.